install-dev.sh

```diff
@@ -1,4 +1,6 @@
 #!/bin/bash
 # called from Pipfile
 pipenv run pip install https://github.com/openwisp/django-netjsonconfig/tarball/master
-pipenv run pip install --upgrade "openwisp-utils[qa]>=0.3.2,<0.4.0"
+pipenv run pip install --upgrade "openwisp-utils[qa]>=0.4.0,<0.5.0"
+# TODO: temporary: remove when openwisp-utils#71 is merged
+pipenv run pip install --upgrade "https://github.com/waleko/openwisp-utils/tarball/issues/40"
```

Comment:
I released openwisp-utils 0.4.1 which contains your patch openwisp/openwisp-utils#71.
You can now remove this temporary addition.